.github/workflows/chipyard-full-flow.yml

```diff
@@ -79,6 +79,7 @@ jobs:
           cd ${{ env.REMOTE_WORK_DIR }}
           eval "$(conda shell.bash hook)"
           export MAKEFLAGS="-j32"
+          mkdir -p $JAVA_TMP_DIR
```

Comment:
Could not having the directory exist cause errors on a user's machine?

Reply:
Unsure yet. I'm testing this in CI but I will do more robust testing later.

Reply:
Looks like that was the case. If this folder didn't exist then SBT couldn't be found. Since the normal CY flow puts the temp directory locally, this should be fine.